src/argilla_sdk/records/_io/_json.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from pathlib import Path
+from typing import List, Union
+
+from argilla_sdk.records._resource import Record
+from argilla_sdk.records._io import GenericIO
+
+
+class JsonIO:
+    @staticmethod
+    def to_json(records: List["Record"], path: Union[Path, str]) -> Path:
```

Comment:
as mentioned before? we will just reason from a path and not allow for passing string? Perhaps this might give issues with encoding etc?